### DISCUSSION TITLE
how to change maximum what i can write that i can make

### DISCUSSION THREAD

**Discussion by irhamdinataa** [austintoddj/canvas#1055] [General]
guys help me how to change the maxium when i write on post , the post cannot saved because the maximum or idk, could you guys help me.

**Comment by irhamdinataa** [comment]
i'm sorry, i just changed data type, and it works -,-

**Comment by austintoddj** [comment]
Closing as inactive / unclear for current product surface.

If you hit a save limit on [v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0), please open a new issue with the exact error (browser network response, PHP `post_max_size` / `upload_max_filesize` if media-related, and Canvas version).
